packages/checkout/widgets-lib/src/components/ChangedYourMindDrawer/ChangedYourMindDrawer.tsx

```diff
@@ -57,6 +58,7 @@ export function ChangedYourMindDrawer({
         >
           <CloudImage
             imageUrl={walletErrorRedUrl}
+            imageResizeServiceUrl={IMAGE_RESIZER_URL[checkout.config.environment]}
```

Comment:
Should we make this property mandatory so that people remember to fill it in? Is there any scenario where we shouldn't be providing this? @glomotion

Reply:
Also minor, but would be nice if you could import the Url already determined instead of having to pick the right one out of the IMAGE_RESIZER_URL object in every component that wants to use it.

Reply:
Ya I was being consistent with our use of the `constants.ts` file..  one of the differences here is that the environment is runtime config during the SDK setup as opposed to ENV variables being set at build time.

Reply:
Could potentially have a bag of this variables in the checkout config.. will give it some thought.

Reply:
Cool neither of these are blockers, can be pondered async from the PR